types/types.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package types
```

Comment:
Not sure if this could go somewhere else, but it's to avoid circular dependency when using it in `executor`

Reply:
Moved into `state` package